### ISSUE TITLE
Bug dans le mail au niveau des pollens

### ISSUE THREAD

**Issue by annepoirot** [betagouv/recosante-mail#91]
Deux bugs : 
- le niveau de risque allergique est écrit en minuscule 

- la majorité du temps il n'y a pas de recommandation

**Comment by fcoufour** [comment]
En commentaire du code source, il est indiqué que dans le cas de l'email, la recommandation n'est donnée que le mercredi et le samedi. Est-ce que ça t'évoque qqch ? https://github.com/betagouv/recosante-api/blob/main/ecosante/recommandations/models.py#L226

Pour le site web, il semble y avoir plusieurs recommandations en rotation pour une même situation  :
- 12 recommandations pour indice RAEP entre 1 et 3
- 3 recommandations pour indice RAEP >= 4
- 1 recommandation pour 0

C'est ce qu'on veut ?

**Comment by annepoirot** [comment]
tiens c'est marrant... Non ca ne me dit rien du tout. Est-ce qu'on aurait voulu mettre ça pour éviter les répétitions ? Ca te dit quelque chose @l-vincent-l ?
J'ai regardé les recommandations et j'en ai passé quelques unes en "publiée", on en a maintenant 24 en tout.

**Comment by annepoirot** [comment]
Effectivement, c'est ce qu'on avait fait avant lorsqu'on avait 1 seule recommandation par jour entre QA et pollen. Donc actuellement il faut enlever le système de jour et mettre le même système que pour l'indice ATMO c'est à dire un choix aléatoire en fonction des filtres.

**Comment by fcoufour** [comment]
Voir PR : https://github.com/betagouv/recosante-api/pull/345
